source/reference/odbc-driver.txt

```diff
@@ -62,8 +62,45 @@ The {+odbc-driver+} is available for:
   .. include:: /includes/temp-warn-odbc-catalina.rst
 
 - Ubuntu 14.04 and 16.04
+
 - RHEL 7
 
+Workaround for Unsupported Platforms 
```

Comment:
Let's keep this more general, that way we will not need to update the values or the instructions if things change. 

I'd suggest something like:

- Install the MySQL Cleartext Authentication plugin for your platform.
- Download and install the Oracle MySQL ODBC driver compatible with your platform from the [MySQL Connector/ODBC Downloads page](https://dev.mysql.com/downloads/connector/odbc/).
- Prior to connecting, ensure that the Cleartext plugin is enabled.